src/containers/profile-page/components/mobile/ProfilePage.tsx

```diff
@@ -320,19 +317,7 @@ const ProfilePage = g(
         )
       } else {
         leftNav = isOwner ? LeftPreset.SETTINGS : LeftPreset.BACK
-        rightNav = isOwner ? (
-          <IconButton
-            className={styles.shareButton}
-            icon={<IconShare />}
-            onClick={() => onShare()}
-          />
-        ) : (
-          <IconButton
-            className={styles.overflowNav}
-            icon={<IconKebabHorizontal />}
-            onClick={onClickOverflow}
-          />
-        )
+        rightNav = <ShareUserButton userId={userId} />
```

Comment:
Are we getting rid of the option to follow/unfollow from the overflow menu?

Reply:
If so, we can remove the onClickOverflow defined above

Reply:
@rickyrombo yeah we are, good call!